index.js

```diff
@@ -29,10 +26,10 @@ module.exports = {
 
 	Errors: require("./src/errors"),
 
-	STRATEGY_ROUND_ROBIN,
-	STRATEGY_RANDOM,
+	RoundRobinStrategy: require("./src/strategies/roundrobin"),
+	RandomStrategy: require("./src/strategies/random"),
```

Comment:
Please put it under a "Strategies" prop like cachers, transporters, serializers with an index.js